x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/EnrichShardMultiSearchAction.java

```diff
@@ -229,11 +231,18 @@ protected MultiSearchResponse shardOperation(Request request, ShardId shardId) t
             final IndexShard indexShard = indicesService.getShardOrNull(shardId);
             try (Engine.Searcher searcher = indexShard.acquireSearcher("enrich_msearch")) {
                 final FieldsVisitor visitor = new FieldsVisitor(true);
+                /*
+                 * Enrich doesn't support defining runtime fields in the search
```

Comment:
you have expanded the comment but not the why :) What made you decide that enrich does not support runtime fields defined in the search request?

Reply:
I didn't so much make the choice as not add it as a new feature. I'll send a new comment.